### ISSUE TITLE
Deprecated get_losses_for function in migration guide

### ISSUE THREAD

**Issue by DorianKodelja** [tensorflow/tensorflow#43341]
## URL(s) with the issue:
https://www.tensorflow.org/guide/migrate?hl=en

## Description of issue (what needs changing):
In the **Custom model_fn with minimal changes** section of the migration guide,  it is recommended to use `get_losses_for` but this function is deprecated and mapped to the model.losses property.
 https://github.com/tensorflow/tensorflow/blob/v2.3.0/tensorflow/python/keras/engine/base_layer.py#L1895

Furthermore, in the example this function is called two times with differents parameters (`None` and `features`) which would results in adding the regularization loss twice.

**Comment by bhack** [comment]
Thanks, Can you submit a PR?

**Comment by DorianKodelja** [comment]
Ok. 
So to be sure, currently, one can simply use `model.losses` ?

**Comment by rmothukuru** [comment]
@MarkDaoust,
Can you please confirm if we can close this issue with respect to [this PR](https://github.com/tensorflow/docs/pull/1678) and with respect to [this Commit](https://github.com/tensorflow/docs/commit/ca65d8e310d33de95f721da6f0405f00ef6a4aed)? Thanks!

**Comment by MarkDaoust** [comment]
Yes, this was fixed by: https://github.com/tensorflow/docs/commit/ca65d8e310d33de95f721da6f0405f00ef6a4aed
